### PR TITLE
Rename `SingleTPUPlugin` to `SingleTPUStrategy`

### PR DESCRIPTION
**Pull request by akihironitta**
## What does this PR do?
Part of #10549.

### Does your PR introduce any breaking changes? If yes, please list them.
Renamed `SingleTPUPlugin` to `SingleTPUStrategy`

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [n/a] Did you make sure to **update the documentation** with your changes? (if necessary)
- [n/a] Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [x] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @justusschock @awaelchli @akihironitta @borda

### CODE REVIEW

**Comment by four4fish** [comment]
Could you also update `Single TPU` in documents and comments please?

**Comment by akihironitta** [comment]
@four4fish All I could find related to `SingleTPU` is only this occurrence bdfbc0d, but are there any other places I should fix?

**Comment by four4fish** [comment]
> @four4fish All I could find related to `SingleTPU` is only this occurrence [bdfbc0d](https://github.com/PyTorchLightning/pytorch-lightning/commit/bdfbc0dfb2d3c7efb70ce82f74b4cfb0d16f4835), but are there any other places I should fix?

Nope, looks good to me! thanks

File: CHANGELOG.md [hunk 1]
```diff
@@ -147,6 +147,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Renamed the `DDPFullyShardedPlugin` to `DDPFullyShardedStrategy` ([#11143](https://github.com/PyTorchLightning/pytorch-lightning/pull/11143))
 
+
+- Renamed `SingleTPUPlugin` to `SingleTPUStrategy` ([#11182](https://github.com/PyTorchLightning/pytorch-lightning/pull/11182))
```
Comment (awaelchli) [inline CHANGELOG.md:151]: Before merging, can you group it under the item above for the TTP rename? See #11123

**Review by awaelchli — APPROVED** [review]

**Review by ananthsub — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -147,6 +148,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Renamed the `DDPFullyShardedPlugin` to `DDPFullyShardedStrategy` ([#11143](https://github.com/PyTorchLightning/pytorch-lightning/pull/11143))
 
+
```
Comment (akihironitta) [inline CHANGELOG.md:151]:
```suggestion
```

**Review by Borda — APPROVED** [review]

**Pull request merged**